test/unit/actions-hubs/CappedTokenTransfersHub.t.sol

```diff
@@ -126,6 +126,13 @@ contract UnitCappedTokenTransfersHub is Test {
   function test_UpdateState_WhenCalledByTheSafe(uint256 _amount) external whenCalledByTheSafe {
     _amount = bound(_amount, 0, cappedTokenTransfersHub.cap(tokens[0]));
 
+    uint256 _currentEpoch =
+      (block.timestamp - cappedTokenTransfersHub.STARTING_TIMESTAMP()) / cappedTokenTransfersHub.EPOCH_LENGTH();
```

Comment:
hmmm, should we do sequential epochs as `1, 2, 3, ...` or should we do step-function epochs such as `1000, 2000, 3000, ...` (each assigned to the starting timestamp)